### PR TITLE
2018 Results page

### PR DESCRIPTION
**Pull request by s-kennedy**
@nifemioluyemi and I put together a results page so that we can show the election results ☑️ 

Here's a little preview of what it looks like:
screenshot_2018-10-21 2018 election results

I've added four columns to `_data/sync/nominees.csv`:
- `Votes` - the number of votes they got
- `Winner` - "Y" if they won, blank if not
- `Percentage`- the percentage of votes they got, without the percentage sign
- `Image` -  an image URL, ideally a square image with their head in the middle (optional) 

So basically we'll just need to fill in that data for the winners and it'll show up on the website!

### CODE REVIEW

**Comment by pnijjar** [comment]
Alternatively we could merge this in and then fix up the data architecture later. But I still feel that populating the fields is a problem, so somebody has to volunteer to do that. 

Because of the time I strongly feel that we should not be trying to populate the headshot field. I understand why we want headshots (and I agree with them in principle) but even if we only get headshots for the winners it creates a lot of extra work for us. So the results have to look good without them.

**Comment by pnijjar** [comment]
I know I have blathered on too much, but I want to mention one more thing: I was also thinking about how the site changes after the election. My original intention was to change the front page with links to the official election results, because the area municipalities are getting them published on their sites. If that assumption is true then there is not as much pressure for us to compile those results.

**Comment by s-kennedy** [comment]
@pnijjar I updated the PR to use the aliases instead of jurisdictions. I also added the municipality name to the Regional Councillor positions, so that should help clarify things. 

I used vote number and percentage because that's how the results are reported on the Region of Waterloo and municipality websites, so I think that's what people expect to see. I think we should just copy the results from official sources (which would mean hard coding the percentages) to ensure that there are no inconsistencies between our results and the official ones. 

The card work with images and without, if we don't have time to add images we just won't do it and it'll still look fine.

Do you have any suggestions for how we should structure the results data instead of putting it in nominees.csv? To me that seems like the easiest way to make the updates. And in 2022, wouldn't we have a new nominees.csv spreasheet anyway? Anyway I think between now and 2022 we'll have enough time to revise the data structure to separate the data by year. 

I put up a live version here with some sample data: https://s-kennedy.github.io/wr-votes/2018-results

So you can see what it looks like with the headshot, without the headshot, with voting numbers and without, etc.

**Comment by pnijjar** [comment]
Yes, maybe my worries about nominees.csv are overblown. I do know that we probably will not be making a lot of changes to the site after the election, although I am going to try and do some cleanup. So I want the data model to be clean now, because I want to be able to reuse this structure if necessary for other elections. 

The "Description" field of aliases.csv is not used anywhere else, so if you want to change the descriptions then that is fine. I do not mind if you add a DisplayTitle column to aliases.csv either.

Thanks for getting rid of jurisdictions. I think this is significantly better.

If we think that the municipalities will publish percentages then that is okay, I guess. 

One thing that was not apparent before I started looking at the site was that you are only publishing winners. Is that your intention? Are we asking people to go to the official sites if they want to see how their favourite non-winning candidate did?

On my computer Jan d'Ailly and Elizabeth Clarke have broken images, because it looks like those images are coming from Twitter. As we populate these image fields they should probably be on our server, not on remote ones. (That, of course, raises copyright issues, but let's leave that aside for now.) I think the proper place for these images will be something like `assets/headshots`. `assets/images/headshots` is another possibility but I think it is a worse one.

**Comment by s-kennedy** [comment]
We were thinking about adding the voting results for all the candidates to the Ward pages as well, like next to the candidate name, but we didn't end up having time to get to it. 

For the images, I'm fine with saving the images to our server, it's just an extra step for whoever's doing it.

**Comment by s-kennedy** [comment]
We were thinking about adding the voting results for all the candidates to the Ward pages as well, like next to the candidate name, but we didn't end up having time to get to it. 

For the images, I'm fine with saving the images to our server, it's just an extra step for whoever's doing it.
